public/app/core/components/QueryOperationRow/QueryOperationRow.tsx

```diff
@@ -164,6 +176,7 @@ const getQueryOperationRowStyles = stylesFactory((theme: GrafanaTheme) => {
       color: ${theme.colors.textBlue};
       margin-left: ${theme.spacing.sm};
       overflow: hidden;
+      text-overflow: ellipsis;
```

Comment:
This is unrelated, just a small fix for `title` used in transformations:

Before:
![Screenshot 2021-10-06 at 16 46 40](https://user-images.githubusercontent.com/1170767/136238342-2b864cbd-8c4e-47aa-8c7b-479f0dc42b49.png)

After:
![Screenshot 2021-10-06 at 16 46 56](https://user-images.githubusercontent.com/1170767/136238376-7c6844d4-7ee2-41b1-9d8e-ab48bf99b914.png)